lib/management/user.ts

```diff
@@ -52,41 +54,61 @@ const withUser = (sdk: CoreSdk, managementKey?: string) => ({
     middleName?: string,
     familyName?: string,
     additionalLoginIds?: string[],
-  ): Promise<SdkResponse<UserResponse>> =>
-    transformResponse<SingleUserResponse, UserResponse>(
-      sdk.httpClient.post(
-        apiPaths.user.create,
-        {
-          loginId,
-          email,
-          phone,
-          displayName,
-          givenName,
-          middleName,
-          familyName,
-          roleNames: roles,
-          userTenants,
-          customAttributes,
-          picture,
-          verifiedEmail,
-          verifiedPhone,
-          additionalLoginIds,
-        },
-        { token: managementKey },
-      ),
+  ): Promise<SdkResponse<UserResponse>>;
+
+  function create(
+    loginId: string,
+    emailOrOptions?: string | UserOptions,
+    phone?: string,
+    displayName?: string,
+    roles?: string[],
+    userTenants?: AssociatedTenant[],
+    customAttributes?: Record<string, AttributesTypes>,
+    picture?: string,
+    verifiedEmail?: boolean,
+    verifiedPhone?: boolean,
+    givenName?: string,
+    middleName?: string,
+    familyName?: string,
+    additionalLoginIds?: string[],
+  ): Promise<SdkResponse<UserResponse>> {
+    const body =
+      typeof emailOrOptions === 'string'
```

Comment:
I wonder if we should add some deprecation message here, let me know what you think

Reply:
Not sure if deprecation, but maybe explain the rational here.

Reply:
added 👍